packages/flutter_tools/lib/src/build_system/targets/ios.dart

```diff
@@ -640,6 +641,11 @@ class ReleaseIosApplicationBundle extends _IosAssetBundleWithDSYM {
           label: buildSuccess ? 'success' : 'fail',
           flutterUsage: environment.usage,
         ).send();
+        environment.analytics.send(Event.appleUsageEvent(
```

Comment:
why is this called appleUsageEvent?  or, why have a single enum for both ios and macos workflows?

Reply:
i made a call to combine the leftover `UsageEvent` [calls](https://docs.google.com/spreadsheets/d/11KJLkHXFpECMX7tw-trNkYSr5MHDG15XNGv6TgLjfQs/edit?resourcekey=0-j4qdvsOEEg3wQW79YlY1-g#gid=0), they were all ungrouped so it may have been the wrong call to combine them all under `appleUsageEvent` instead of separate `iosUsageEvent` and `macosUsageEvent`.

We spoke offline about this but if you think this won't scale well if we have additional apple related events, i can definitely revert the original PR in dart-lang/tools

Reply:
it's not a big deal, it just struck my eye as unusual. there has also been discussion about what to name the union of macOS and iOS in the codebase (many places in the tool we call it "Darwin", but others "Apple" ). let's leave this.

Reply:
gotcha sounds good! i'll keep this in mind for future work